.github/workflows/go.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         go-version: '1.22'
 
     - name: Build
-      run: go build -v ./...
+      run: go build -tags integration -v ./...
 
     - name: Test
-      run: go test -v ./...
+      run: go test -v -tags integration ./...
```

Comment:
Also, this way you'll run integration tests together with your unit tests.
Maybe it's better to separate them completely? 

Proofs that unit tests are not skipped:
<img width="1033" alt="image" src="https://github.com/GenesisEducationKyiv/software-engineering-school-4-0-Hukyl/assets/93580374/71b7382d-ec7f-4337-8fcd-28e45e82c4c2">
<img width="1072" alt="image" src="https://github.com/GenesisEducationKyiv/software-engineering-school-4-0-Hukyl/assets/93580374/678b61cf-07c5-4693-965a-8a79e25ad500">

Reply:
to make them work completely separate Andrii needs to add both `//go:build integration` to integration tests and `//go:build !integration` to unit tests, right?
@fabl3ss @KushnerykPavel  what do you think?